packages/@aws-cdk/aws-apigateway/lib/apigatewayv2.ts

```diff
@@ -869,7 +867,6 @@ export class CfnAuthorizerV2 extends cdk.CfnResource implements cdk.IInspectable
    *
    * @param inspector - tree inspector to collect and process attributes
    *
-   * @stability experimental
    */
   public inspect(inspector: cdk.TreeInspector) {
```

Comment:
@nija-at any idea what's this about? Why is this method in a non `.generated` file?

Reply:
see commit description - https://github.com/aws/aws-cdk/commit/d58667e473c1e07ac8f2f073b3a541569a014e01